lib/statement_timeout.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module StatementTimeout
+  module QueryMethodsExtension
+    def statement_timeout(timeout)
```

Comment:
Would be cool to eventually support usage without a block, too. E.g.:

```rb
License.statement_timeout(1.minute).some_heavy_report
```

I reserved the `statement_timeout` gem name, too.